### PR TITLE
Making setInterval optional

### PR DESCRIPTION
**Pull request by goyalvarun601**
@kdimatteo

### CODE REVIEW

File: dist/jquery.audio-plugin.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
   'use strict';
 
-  var MHEAudio = function(el){
+  var MHEAudio = function(el, setInterval){
```
Comment (kdimatteo) [inline dist/jquery.audio-plugin.js]: Can we use something else since `setInterval` is a JS Internal method?

File: dist/jquery.audio-plugin.js [hunk 2]
```diff
@@ -26,9 +26,13 @@
     this.inError        = false;
     this.timerLimit     = 10;
 
+    this.$el.append('<span class="error__message">Still processing. Please try again.</span>');
```
Comment (kdimatteo) [inline dist/jquery.audio-plugin.js]: :+1:

File: dist/jquery.audio-plugin.js (hunk 1 above)
Comment (goyalvarun601) [inline dist/jquery.audio-plugin.js]: how about addInterval or useInterval?

**Comment by goyalvarun601** [comment]
Made it to addInterval.

**Comment by kdimatteo** [comment]
:+1:

**Pull request merged**